04_verify.sh

```diff
@@ -311,4 +311,5 @@ done
 
 
 echo -e "\nNumber of failures : $FAILS"
+. "$(dirname "$(readlink -f "${0}")")"/scripts/fetch_manifests.sh
```

Comment:
At this moment we haven't deployed any target cluster so essentially no metal3 objects, so what would it fetch?